processing/multivariate_comparison.py

```diff
@@ -938,15 +940,21 @@ def create_slope_status(self, x, y, year, slope, mean_slope, fig_loc,
                               f'{delta_s}% is greater than error '
                               f'threshold {delta_s_error_percent}%')
                 log_obj.error(status_msg)
+                self.deviated_slopes.append(round(float(mean_slope-slope), 3))
+                self.deviated_years.append(int(year))
             elif abs(delta_s) > delta_s_warning_percent:
                 status_msg = (f'Slope: {slope}; slope deviation '
                               f'{delta_s}% is greater than warning '
                               f'threshold {delta_s_warning_percent}%')
                 log_obj.warning(status_msg)
+                self.deviated_slopes.append(round(float(mean_slope-slope), 3))
+                self.deviated_years.append(int(year))
             else:
                 status_msg = (f'Slope: {slope}; slope deviation: '
                               f'{delta_s}%')
                 log_obj.info(status_msg)
+                self.deviated_slopes.append(round(float(mean_slope-slope), 3))
```

Comment:
Repeated operations should be abstracted

Reply:
I was wondering if the else case should append the difference into deviated_slopes? I'm not sure if the else case is noting deviations above a certain threshold unlike the if and elif case

Reply:
Where is `self.deviated_slopes` used?

Reply:
OK, not entirely sure. N/A may be a useful text in this case, but need to check with Housen

Reply:
In the summary statistics for the Slope of Fit graphs for multivariate (noting the years and slopes that have relatively deviated regression slopes)